Formula/burp.rb

```diff
@@ -13,14 +17,51 @@ class Burp < Formula
     sha256 "d3b8cc95839835dd9ba9574444d5cf7394d398376fe2a06fcd3ba9a00b395476" => :mavericks
   end
 
+  devel do
+    url "https://github.com/grke/burp/archive/2.0.40.tar.gz"
+    sha256 "8180b87321e8dd664a23d17cba25406ad3339a73e06bfdde9ec703e32ebe11f2"
+
+    depends_on "autoconf" => :build
+    depends_on "automake" => :build
+    depends_on "libtool" => :build
+
+    resource "uthash" do
+      url "https://github.com/troydhanson/uthash.git",
+        :revision => "22646fcb7ce80be08d8917e153dabb272476c710"
+    end
+  end
+
+  head do
+    url "https://github.com/grke/burp.git"
+
+    depends_on "autoconf" => :build
+    depends_on "automake" => :build
+    depends_on "libtool" => :build
+
+    resource "uthash" do
+      url "https://github.com/troydhanson/uthash.git"
+    end
+  end
+
   depends_on "librsync"
   depends_on "openssl"
 
-  # patches to change directories to brew conventions in Makefile and config
-  # files
-  patch :DATA
-
   def install
+    if build.devel? || build.head?
```

Comment:
I'd just feel free to use `unless build.stable?` here, if you wanted to. Where it applies to both devel & head `unless build.stable?` is a valid syntax choice.